gsms.py

```diff
@@ -205,5 +304,38 @@ def main() -> None:
                                 'custom location.')
 
 
+def get_win_path(folderid, user_handle=UserHandle.current):
```

Comment:
- Change default `user_handle` to `wintypes.HANDLE(0)`... add type hints and return type.
- `UserHandle` class is not needed.
- `folderid` to `folder_id`

```py
def get_win_path(folder_id: str, user_handle: str = wintypes.HANDLE(0)) -> str:
```

Reply:
`user_handle` has been removed as the class was removed. Just directly using `wintypes.HANDLE(0)` in the function call of `_SHGetKnownFolderPath` now